### PR TITLE
279 differentiate enemies projectiles

### PR DESCRIPTION
**Pull request by 31Nathan**
We now have different types of projectiles
Electric is fast, doing cheap damage
Fire is slow, doing heavy damage
Ice has a huge damage radius
Basic is still Basic ^^

### CODE REVIEW

File: Client/ClientRoom.hpp [hunk 1]
```diff
@@ -75,6 +76,15 @@ namespace client_data
         /// @brief Hold a room connection process
         void _connectToARoom();
 
+        /// @brief Init all the sprites needed for the player
```
Comment (LucasTesnier) [inline Client/ClientRoom.hpp:79]: missing param documentation (same below) :)

**Review by LucasTesnier — CHANGES_REQUESTED** [review]
Missing param documentation three times, otherwise it's okay for me

File: libs/R-TypeLogic/EntityManipulation/CreateEntitiesFunctions/CreateEnemyProjectile.cpp [hunk 2]
```diff
@@ -18,14 +70,15 @@ namespace ecs
 
         Entity &entity = world.addEntity();
         auto guard = std::lock_guard(entity);
+        unsigned short parentType = enemy->getComponent<Enemy>().enemyType;
+
+        switch (parentType) {
```
Comment (ZQUEMA) [inline libs/R-TypeLogic/EntityManipulation/CreateEntitiesFunctions/CreateEnemyProjectile.cpp:75]: I love it

**Review by ZQUEMA — APPROVED** [review]
Do the change asked by @LucasTesnier and it's ok

**Review by LucasTesnier — APPROVED** [review]

**Review by TomDUVAL-MAHE — APPROVED** [review]

**Pull request merged**